backends/graphs/README.md

```diff
@@ -23,6 +25,43 @@ cd out
 dot <name>.dot -Tpng > <name>.png
 ```
 
+For generation of dot fullGraph, use option `--fullGraph` and 
+for generation of fullGraph represented in json, use option `--jsonOut`.
+
+## Format of json output
```

Comment:
Are there any tools that consume this json?

Reply:
There are no tools that consume this json. 
I just have a python script that generates the fullGraph in dot format from this json input. But it is the same fullGraph as the one generated from *p4c-graphs* backend.

The idea behind fullGraph in json was that it could be manipulated more easily than fullGraph in dot, and used for further processing of this graph. But no such tools were implemented yet.

Reply:
I uploaded the script here [https://gist.github.com/st22nestrel/4d23d028b0b211c91e86a194c1269e28]()